src/Function/Construct/Composition.agda

```diff
@@ -204,28 +204,85 @@ module _ {R : Setoid a ℓ₁} {S : Setoid b ℓ₂} {T : Setoid c ℓ₃} where
 ------------------------------------------------------------------------
 -- Propositional bundles
 
-infix 8 _∘-⟶_ _∘-↣_ _∘-↠_ _∘-⤖_ _∘-⇔_ _∘-↩_ _∘-↪_ _∘-↔_
+infix 8 _⟶-∘_ _↣-∘_ _↠-∘_ _⤖-∘_ _⇔-∘_ _↩-∘_ _↪-∘_ _↔-∘_
 
-_∘-⟶_ : (A ⟶ B) → (B ⟶ C) → (A ⟶ C)
-_∘-⟶_ = function
+_⟶-∘_ : (A ⟶ B) → (B ⟶ C) → (A ⟶ C)
+_⟶-∘_ = function
 
-_∘-↣_ : A ↣ B → B ↣ C → A ↣ C
-_∘-↣_ = injection
+_↣-∘_ : A ↣ B → B ↣ C → A ↣ C
+_↣-∘_ = injection
 
-_∘-↠_ : A ↠ B → B ↠ C → A ↠ C
-_∘-↠_ = surjection
+_↠-∘_ : A ↠ B → B ↠ C → A ↠ C
+_↠-∘_ = surjection
 
-_∘-⤖_ : A ⤖ B → B ⤖ C → A ⤖ C
-_∘-⤖_ = bijection
+_⤖-∘_ : A ⤖ B → B ⤖ C → A ⤖ C
+_⤖-∘_ = bijection
 
-_∘-⇔_ : A ⇔ B → B ⇔ C → A ⇔ C
-_∘-⇔_ = equivalence
+_⇔-∘_ : A ⇔ B → B ⇔ C → A ⇔ C
+_⇔-∘_ = equivalence
 
-_∘-↩_ : A ↩ B → B ↩ C → A ↩ C
-_∘-↩_ = leftInverse
+_↩-∘_ : A ↩ B → B ↩ C → A ↩ C
+_↩-∘_ = leftInverse
 
-_∘-↪_ : A ↪ B → B ↪ C → A ↪ C
-_∘-↪_ = rightInverse
+_↪-∘_  : A ↪ B → B ↪ C → A ↪ C
+_↪-∘_ = rightInverse
 
-_∘-↔_ : A ↔ B → B ↔ C → A ↔ C
-_∘-↔_ = inverse
+_↔-∘_ : A ↔ B → B ↔ C → A ↔ C
+_↔-∘_ = inverse
+
+
+------------------------------------------------------------------------
+-- DEPRECATED NAMES
+------------------------------------------------------------------------
+-- Please use the new names as continuing support for the old names is
+-- not guaranteed.
+
+-- Version 1.7
+
+_∘-⟶_ = _⟶-∘_
+{-# WARNING_ON_USAGE _∘-⟶_
+"Warning: _∘-⟶_ was deprecated in v1.7.
```

Comment:
Perfect, except that it should be `v2.0` everywhere, not `v1.7` :smile:

Reply:
Oh my bad I will correct it 😄